File: src/main/java/com/monis/Hello.java
package com.monis;

import com.monis.student.Student;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello");
//        Person p = new Person();
//        Person p = new Person(66.5f, 1.7f);
        Person p = new Person("monis",66.5f, 1.7f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());

        int score = 88;
        System.out.println(score >= 60 && score < 90);
        System.out.println(score < 80 || score > 90);
        char c = 'A';
        System.out.println(c > 60);
        System.out.println(c > 'a');

        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.length());
        System.out.println(s.substring(3));
        System.out.println(s.substring(1,4));

        String s1 = null;

       /* int age = 19;
        Integer age2 = 19;
        age2.toString();
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        double height = 1.7;
        boolean adult = true;
        String name = "monis";*/

    }
}

File: src/main/kotlin/com/monis/Hello.kt
package com.monis

fun main(args: Array<String>) {
//    println("Hello Kotlin")
//    Human().hello()
//    val h = Human().hello()
//    h.hello()
//    val h = Human(66.5f,1.7f)
    val h = Human(weight = 66.5f,height = 1.7f)
    println(h.bmi())
    val score = 88
    println(score >80)
    var c: Char = 'A'
    println(c.toInt() > 60)

    var s : String? = "abcde"
    s = null
//    println(s!!.length)
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))


   /* val age : Int = 19
    var weight = 66.5
    var height = 1.7f
    var name : String
    name = "monis"*/
}

class Human(var name:String = "", var weight:Float,var height:Float){
    init {
        println("test $weight")
    }
//    constructor(name:String,weight:Float,height: Float):this(weight,height)

    fun bmi(): Float {
        val bmi = weight / (height*height)
        return bmi
    }

    fun hello() {
        println("Hello Kotlin")
    }
}